.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we add the [bracketSpacing](https://prettier.io/docs/en/options.html#bracket-spacing) rule? What do you think?

Reply:
Added 👍